project/ConsoleTracker/ConsoleTracker/urls.py

```diff
@@ -22,7 +22,10 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('timer/', views.timer, name = 'timer'),
+    path('time_manager/<int:id>', views.time_manager, name = 'time_manager'),
+    path('timer/<int:id>', views.timer, name = 'timer'),
```

Comment:
How is this put to use? Is it attached to any model?

Reply:
@erick576 Yeah it's attached to the Machine's id field that's added automatically added by Django.

Reply:
@JesseyH are they attached to anything though? I know there a part for the machines that display all objects on the page but what models in **models.py** are "**time_manager**" and "**timer**" referencing since you have an id path for them? I tried with the timestamp model but it didn't work

Reply:
What do you mean by attached? the ID refers to the Machine Model. Each time_manager and timer page is specific to the machine it is referencing (via the ID). You also need to create at least one User, because I hard coded the pages to use the User ID with the ID 1